src/components/EmbeddedViewer/EmbeddedViewer.js

```diff
@@ -21,7 +20,6 @@ const EmbeddedViewer = () => {
   return loading === "success" ? (
     <div className="embedded-viewer">
       <Output />
-      <RunnerControls />
```

Comment:
🔧 does it make more sense for the `RunBar` to be here for the embedded viewer rather than adding logic to `Output` to see if it's embedded?

Reply:
Well... 😅 If you have it outside the `Output` it messes with the styling. It was okay before because we were just floating the button above the `Output` anyway, but now it has to look like it's inside that container